pylds/tools.py

```diff
@@ -46,12 +46,12 @@ def draw_lagrangian_descriptor(LD, LD_type, grid_parameters, tau, p_value, norm
         LD = LD / LD.max()  # Scale LD output
```

Comment:
`docstrings` need an update describing the corresponding enhancements made.

Reply:
Now that you point it out, the docstrings in tools need an overhaul. Will do.